Python/ceval_gil.c

```diff
@@ -205,8 +200,8 @@ static void destroy_gil(struct _gil_runtime_state *gil)
     COND_FINI(gil->switch_cond);
     MUTEX_FINI(gil->switch_mutex);
 #endif
-    _Py_atomic_store_explicit(&gil->locked, -1,
-                              _Py_memory_order_release);
+    _Py_atomic_store_int_relaxed(&gil->locked, -1);
+    _Py_atomic_store_int_release(&gil->locked, -1);
```

Comment:
Should only have the `_Py_atomic_store_int_release` call

Reply:
Thanks, this is my mistake.